.ci/Jenkinsfile

```diff
@@ -130,7 +127,7 @@ node('Linux') {
                                     def command = "export PYENV_ROOT=${pyenvdir} && " \
                                                 + "export PATH=\"${pyenvdir}/versions/\${pythonVersion}/bin:${pyenvdir}/bin:\$PATH\" && " \
                                                 + "pyenv global ${pythonVersion} && " \
-                                                + "sudo /home/conan/select_cmake.sh 3.16.9"
+                                                + "sudo /home/conan/select_cmake.sh 3.23.5"
```

Comment:
I am a bit concerned that this would mean that some of our examples do not work with older CMake, but 🤞